htmd/molecule/voxeldescriptors.py

```diff
@@ -4,12 +4,13 @@
 # No redistribution in whole or part
 #
 from htmd.molecule.util import boundingBox
-import numpy as np
```

Comment:
Never trust pycharm on unused dependencies. This file will crash. The last function uses `np.`